CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-x.x.x Release notes (yyyy-MM-dd)
```

Comment:
This line should be copied down to right before the new changes start (e.g. `* Expose additional authentication-related errors that might be reported by...`), and then reverted, I think. The 2.10.0 changelog entry will need a new `Enhancements` header as well.